packages/assets-controllers/src/assetsUtil.ts

```diff
@@ -292,3 +293,68 @@ export async function reduceInBatchesSerially<
   const finalResult = workingResult as Result;
   return finalResult;
 }
+
+/**
+ * Maps an OpenSea V2 NFT to the V1 schema.
+ * Should work for both 'get_nft' and 'list_nfts_by_account' responses.
+ * @param nft - The V2 NFT to map.
+ * @returns The NFT in the V1 schema.
+ */
+export function mapOpenSeaNftV2ToV1(nft: any): ApiNft {
+  return {
+    token_id: nft.identifier,
+    num_sales: null,
+    background_color: null,
+    image_url: nft.image_url,
+    image_preview_url: nft.image_url,
+    image_thumbnail_url: nft.image_url,
+    image_original_url: null,
+    animation_url: nft.animation_url,
+    animation_original_url: null,
+    name: nft.name,
+    description: nft.description,
+    external_link: null,
+    asset_contract: {
+      address: nft.contract,
+      asset_contract_type: null,
+      created_date: null,
+      schema_name: nft.token_standard?.toUpperCase(),
```

Comment:
It seems like based on the spec, this will always be present, so is there a need for `?.`?

Reply:
This field is listed in `required`, and I think I've always seen it present.  I have seen one field ("next") listed as required but it actually omitted though, so not 100% trusting of the spec.